zazu/repo/commands.py

```diff
@@ -81,10 +87,62 @@ def clone(ctx, repository_url, nohooks, nosubmodules):
 
 
 @repo.command()
+@click.option('--nohooks', is_flag=True, help='does not install git hooks in the repo')
 @click.pass_context
-def init(ctx):
+def init(ctx, nohooks):
     """Initialize repo directory structure"""
-    raise NotImplementedError
+    def _zazu_yaml(issue_tracker={}, stylers=[]):
+        """builds zazu.yaml file"""
+        if issue_tracker:
+            zazu_yaml_obj = issue_tracker
+        else:
+            zazu_yaml_obj = {}
+        if stylers:
+            zazu_yaml_obj['style'] = {key: {'options': [None]} for key in stylers}
```

Comment:
Right, il leave it empty. Kind of a fine line between being useful and being a baby sitter.